src/main/resources/db/testdata/insert-testdata.sql

```diff
@@ -20,11 +20,19 @@ INSERT INTO room_type (description, code) VALUES ('Conference Room','conference-
 
 insert into session_type(code, description)
 values ('K-Fast Track and Applications', 'K-Fast Track and Applications');
+insert into session_type(code, description)
+values ('Without case and hearing type', 'Without case and hearing type');
 
 insert into case_type(code, description)
 values('K-Fast Track', 'K-Fast Track');
 insert into case_type(code, description)
 values('K-Small Claims', 'K-Small Claims');
+insert into case_type(code, description)
+values('Without hearing type', 'Without hearing type');
+insert into case_type(code, description)
+values('Matching only session type', 'Matching only session type');
```

Comment:
If I understand correctly these should be named like "linked only ...", because the word matching is a little bit confusing for the others.

Reply:
changed, thanks